MaterialLib/Component/ComponentProperties.h

```diff
@@ -0,0 +1,55 @@
+/**
+ * \file
+ *
+ * \copyright
+ * Copyright (c) 2012-2018, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ */
+
+#pragma once
+
+#include <Eigen/Dense>
+#include <memory>
+#include <vector>
```

Comment:
vector and Eigen includes are not needed in this file.

Reply:
Removed.